### PR TITLE
[v6r20] RSS Minor fixes

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*RSS
FIX: Mostly style changes to RSS

ENDRELEASENOTES

### CODE REVIEW

File: ResourceStatusSystem/Client/ResourceManagementClient.py [hunk 1]
```diff
@@ -4,15 +4,18 @@
 
 """
 
-__RCSID__ = '$Id:  $'
+__RCSID__ = '$Id$'
 
 from DIRAC.Core.DISET.RPCClient import RPCClient
 
 # a method that makes the first letter uppercase only (and leaves the rest letters unaffected)
```
Comment (andresailer) [inline ResourceStatusSystem/Client/ResourceManagementClient.py]: This should be the docstring for the `uppercase_first_letter` function

File: ResourceStatusSystem/Client/ResourceStatusClient.py [hunk 2]
```diff
@@ -4,24 +4,25 @@
...
-from DIRAC.ConfigurationSystem.Client.Helpers.Operations    import Operations
-from DIRAC.FrameworkSystem.Client.NotificationClient        import NotificationClient
+from DIRAC.ConfigurationSystem.Client.Helpers.Operations import Operations
+from DIRAC.FrameworkSystem.Client.NotificationClient import NotificationClient
 
 
-__RCSID__ = '$Id:  $'
+__RCSID__ = '$Id$'
 
 # a method that makes the first letter uppercase only (and leaves the rest letters unaffected)
```
Comment (andresailer) [inline ResourceStatusSystem/Client/ResourceStatusClient.py]: Docstring for `uppercase_first_letter` function

File: ResourceStatusSystem/Client/ResourceStatusClient.py [hunk 3]
```diff
@@ -50,16 +45,6 @@ class ResourceStatusClient( object ):
...
   """
 
-  def __init__( self ):
-    '''
-      The client tries to connect to :class:ResourceStatusDB by default. If it
-      fails, then tries to connect to the Service :class:ResourceStatusHandler.
-    '''
-
-    self.rssService = RPCClient( "ResourceStatus/ResourceStatus" )
-    self.validElements = RssConfiguration.getValidElements()
```
Comment (atsareg) [inline ResourceStatusSystem/Client/ResourceStatusClient.py]: This initialization is not needed any more ?

Comment (fstagni) [reply ResourceStatusSystem/Client/ResourceStatusClient.py]: Seems not to me.

File: ResourceStatusSystem/Client/ResourceManagementClient.py [hunk 4]
```diff
@@ -4,15 +4,18 @@
...
+__RCSID__ = '$Id$'
 
 from DIRAC.Core.DISET.RPCClient import RPCClient
 
-# a method that makes the first letter uppercase only (and leaves the rest letters unaffected)
+
 def uppercase_first_letter(key):
+  """ a method that makes the first letter uppercase only (and leaves the remaining letters unaffected)
+  """
   return key[0].upper() + key[1:]
```
[the remark this answers is not in the document]
Comment (fstagni) [reply ResourceStatusSystem/Client/ResourceManagementClient.py]:
```
n [1]: 'pippo'.capitalize()
Out[1]: 'Pippo'

In [2]: 'pippO'.capitalize()
Out[2]: 'Pippo'
```
while I want `pippO` --> `PippO`

File: ResourceStatusSystem/Client/ResourceManagementClient.py [hunk 5]
```diff
@@ -90,11 +85,10 @@ def selectAccountingCache( self, name = None, plotType = None, plotName = None,
     :return: S_OK() || S_ERROR()
     '''
 
-    return self.rmService.select( 'AccountingCache', self._prepare(locals()) )
-
+    return RPCClient("ResourceStatus/ResourceManagement").select('AccountingCache', self._prepare(locals()))
```
[the remark this answers is not in the document]
Comment (fstagni) [reply ResourceStatusSystem/Client/ResourceManagementClient.py]: I would do it if the code of this ResourceManagementClient could inherit from `Client`, but it doesn't...

File: ResourceStatusSystem/Client/ResourceManagementClient.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (chaen) [reply ResourceStatusSystem/Client/ResourceManagementClient.py]: Ah ok ! Did not think of that

File: ResourceStatusSystem/Client/ResourceManagementClient.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (chaen) [reply ResourceStatusSystem/Client/ResourceManagementClient.py]: Ah? What prevent's it ?

[the remark this answers is not in the document]
Comment (fstagni) [reply ResourceStatusSystem/Client/ResourceManagementClient.py]: there's this `self._prepare(locals())` everywhere (not that I like it...)

[the remark this answers is not in the document]
Comment (chaen) [reply ResourceStatusSystem/Client/ResourceManagementClient.py]: O_o why so much hate...

Anyway, you can still do 
`return self._getRPC().select(...)`

Or am I missing something ?

[the remark this answers is not in the document]
Comment (fstagni) [reply ResourceStatusSystem/Client/ResourceManagementClient.py]: `_getRPC()` just does `RPCClient()`: https://github.com/DIRACGrid/DIRAC/blob/integration/Core/Base/Client.py#L92

[the remark this answers is not in the document]
Comment (chaen) [reply ResourceStatusSystem/Client/ResourceManagementClient.py]: yes, but this allows to have the RPCClient circumvented to the Client class, and ease the potential migration to https

**Comment by fstagni** [comment]
@atsareg please merge this one too. It contains some bug fixes and I don't have time right now to go through Chris' (correct) remark.

**Pull request merged**